Upgrading mix to 6.0.16 [webpack-cli] Error [ERR_REQUIRE_ESM]: Must use import to load ES Module

-   Laravel Mix Version: 6.0.16 (`npm list --depth=0`)
-   Node Version (`node -v`): 14.16.1
-   NPM Version (`npm -v`): 7.10.0
-   OS: Ubuntu 16.04.7

### Description:

Upgrading app from mix 2.1.14 to 6.0.16 is not working. (while upgrading from Laravel 5.7 to 8.0)

### Steps To Reproduce:

This is my package.json:

{
    "private": true,
    "scripts": {
        "development": "mix",
        "watch": "mix watch",
        "watch-poll": "mix watch -- --watch-options-poll=1000",
        "hot": "mix watch --hot",
        "production": "mix --production"
    },
    "devDependencies": {
        "@fortawesome/fontawesome-pro": "^5.15.2",
        "axios": "^0.21.1",
        "bootstrap": "^4.6.0",
        "cross-env": "^5.1",
        "jquery": "^3.5.1",
        "laravel-mix": "^6.0.16",
        "lodash": "^4.17.21",
        "popper.js": "^1.16.1",
        "sass": "^1.32.11",
        "vue": "^2.5.7"
    },
    "dependencies": {
        "mobile-menu": "^2.0.8",
        "postcss": "^8.2.10",
        "strip-ansi": "^7.0.0"
    }
}

npm install laravel-mix@latest
changing the scripts in package.json as suggested in the documentation
npm run production':

[webpack-cli] Error: Cannot find module 'strip-ansi'

So naturally I run npm install strip-ansi and re-run npm run production.

Then I get:

[webpack-cli] Error [ERR_REQUIRE_ESM]: Must use import to load ES Module: /fiveeight/node_modules/strip-ansi/index.js require() of ES modules is not supported.

Anyone know what might be happening? Also anyone know why npm didn't just satisfy it's need of strip-ansi? this upgrade has been a little odd.

--- a/package.json
+++ b/package.json
@@ -83,6 +83,7 @@
         "postcss-load-config": "^3.0.0",
         "postcss-loader": "^5.2.0",
         "semver": "^7.3.4",
+        "strip-ansi": "^6.0.0",
         "style-loader": "^2.0.0",
         "terser": "^5.3.7",
         "terser-webpack-plugin": "^5.0.0",

